db/migrate/20160901010430_change_column_type.rb

```diff
@@ -1,13 +0,0 @@
-class ChangeColumnType < ActiveRecord::Migration
```

Comment:
I think the first one added it as a string, but this one switched it over to be a float.  I'm pretty sure both are needed?  Thoughts, @mailethiesen??